README.md

```diff
@@ -123,23 +123,21 @@ If the auto-inject flag is disabled, the tooling plugin will have to be applied
 Gradle DSL
 ```groovy
 plugins {
-  id 'com.squareup.tooling' version '0.1.0'
+  id 'com.squareup.tooling' version '0.1.1'
 }
 ```
 
 Legacy
 ```groovy
 buildscript {
   dependencies {
-    classpath "com.squareup.affected.paths:tooling-support:0.1.0"
+    classpath "com.squareup.affected.paths:tooling-support:0.1.1"
   }
 }
 
 apply plugin: "com.squareup.tooling"
 ```
 
-**WARNING:** Do not apply the plugin if the auto-inject flag is enabled. The analysis will fail due to the same plugin being applied twice.
```

Comment:
This warning wasn't needed any longer. There is a check done to prevent this issue.